kong/plugins/http-log/handler.lua

```diff
@@ -189,12 +193,43 @@ function HttpLogHandler:log(conf)
 
   local queue_conf = Queue.get_plugin_params("http-log", conf, make_queue_name(conf))
   kong.log.debug("Queue name automatically configured based on configuration parameters to: ", queue_conf.name)
+  local entry = cjson.encode(kong.log.serialize())
+
+  if queue_conf.max_batch_size == 1 then
```

Comment:
This seems to be a hidden usage? No doc.

Reply:
This behavior is currently undocumented.